src/Components/SideDrawer.jsx

```diff
@@ -101,7 +101,11 @@ export default function SideDrawerWrapper() {
       openDrawer()
       turnCommentsOn()
     }
-  }, [location, openDrawer, setSelectedIssueId, turnCommentsOn])
+    // This address bug #314 by clearing selected issue when new model is loaded
+    if (issueHash === undefined && isDrawerOpen) {
+      setSelectedIssueId(null)
```

Comment:
it clears the selected note and moves the navpanel to the list view, on model reload.